code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/fix-issues-in-a-scan-result.adoc

```diff
@@ -0,0 +1,31 @@
+:topic_type: task
+
+[.task]
+
+== Fix Issues in a Scan Result
+
+On *Projects*, you can fix scan results across all code categories.
+
+[.procedure]
+
+. Access scan results on *Projects*.
+
+.. Select a code category with an issue.
++
+In this example, on IaC Misconfiguration you see `aws_s3_bucket.bad_bucket` resource block with an issue that has a fix.
++
+image::proj-22.png[width=800]
+
+. Create a Pull Request with an issue fix.
+
+.. Select add corresponding to an issue and then view the issue in fix cart.
++
+In this example, you see `aws_s3_bucket.bad_bucket` in the fix cart.
++
```

Comment:
we should show in the documentation the fact that you click on an issue and then you see in the sidecar (resource explorer) the issue that was detected and the suggested fix

Reply:
Added this section. @GiladMark